index.js

```diff
@@ -315,6 +325,13 @@ async function start() {
     } else {
       console.log("You can't drop what you don't have!");
     }
+    // 'read' action
```

Comment:
this type of comment is helpful; it tells me what to expect when the player enters the 'read' action